urnai/actions/chain_of_actions_base.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC
+
+from action_base import ActionBase
+
+from urnai.utils.returns import List
+
+
+class ChainOfActionsBase(ABC):
+
+	"""This class works as a sequence of actions"""
+	
+	action_list: List[ActionBase]
+	
+	def __init__(self):
+		self.action_list = []
+	
+	def next(self):
+		# Executing the next action in the sequence
+		
+		if self.get_len() == 0: 
+			return None
+	
+		first_action = self.action_list[0]
+		
+		if first_action.is_complete():
+			self.action_list.pop(0)
+			self.next()
+		else:
+			first_action.run()
+	
+	def check(self) -> bool:
+		# Checking all of the actions in the sequence
```

Comment:
The comment about a function must be made using `'''` or `"""`, otherwise Python will not recognize it in the `help` function.